src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -72,6 +72,13 @@ public UniquePersonList(UniquePersonList source) {
         internalList.addAll(source.internalList);
     }
 
+    /**
+     * Returns an modifiable java List view.
+     */
+    public List<Person> mutableListView() {
+        return internalList;
```

Comment:
Returning the `internalList` and sorting it would mean when you use the `list` command , the list would be already sorted. Could return a new list instead.

Reply:
noted, thanks! 👍

Reply:
I think this is a fine feature actually.